scipy/stats/tests/test_multivariate.py

```diff
@@ -1635,6 +1637,144 @@ def test_haar(self):
         res = kstest(x.ravel(), uniform(-np.pi, 2*np.pi).cdf)
         assert_(res.pvalue > 0.05)
 
+
+class TestMultivariateHypergeom(object):
```

Comment:
general comment for this testing class; there is some repeated computation on the distribution. It might make sense to use a fixture so that you don't need to instantiate the same distribution object so many times.

To be clear not everything is repeated but if it's possible to either reduce via pytest fixture or alternately via the `pytest.mark.parametrize` annotation here that might make it less lines of code but still easy to follow.

Consider this comment a (non-blocking) suggestion.

Reply:
fixed.

Reply:
ok, I see them in the methods inside this test class, marking resolved, thanks @tirthasheshpatel